docs/user/adding-glean-to-your-project.md

```diff
@@ -262,16 +317,42 @@ When re-enabling, core Glean metrics will be recomputed at that time.
 
 <div data-lang="Python" class="tab">
 
-> **Note**: This content is a placeholder.  The Python bindings are under development.
-
 ### Initializing the Glean SDK
 
+The main control for Glean is on the `glean.Glean` singleton.
+
+The Glean SDK should be initialized as soon as possible, and importantly, before any other libraries in the application start using Glean.
+Library code should never call `Glean.initialize`, since it should be called exactly once per application.
+
+
 ```python
-import glean
-cfg = glean.Configuration()
-glean.Glean.initialize(cfg, "/path/to/datadir")
+from glean import Glean
+
+# Call Glean.set_upload_enabled first, since Glean.initialize might send pings
+# if there are any metrics queued up from a previous run.
+Glean.set_upload_enabled(True)
+
+Glean.initialize(
```

Comment:
What do you think of making the argument keyword argument only to be extra explicit?

Reply:
That's a great idea.  I filed a [follow-on bug for that](https://bugzilla.mozilla.org/show_bug.cgi?id=1597703) -- just to keep this one more-or-less docs only.